magic_example.ipynb

```diff
@@ -53,12 +52,17 @@
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "import tutorial.reveal_solution as rs\n",
```

Comment:
This didn't work for me when I was trying.

Reply:
It was removed in a later commit, let's keep feature branches isolate. Here we only show the output, revealing solutions is for another branch.